.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "targets": ["last 2 chrome versions", "last 2 firefox versions"],
+  "plugins": [
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "regenerator": true
+      }
+    ]
+  ],
```

Comment:
a few notes about that (all optional for this patch):
1. my understanding is that `@babel/preset-env` will add this plugin automatically
2. `true` seems to be the default for the `regenerator` option
3. That said I doubt this will even be used if you target latest versions of chrome and firefox. :)